code/game/machinery/doors/poddoor.dm

```diff
@@ -56,6 +58,17 @@
 			else if(W.use_tool(src, user, 10 SECONDS, volume=50))
 				deconstruct(TRUE)
 
+	if(W.tool_behaviour == TOOL_DECONSTRUCT)
+		if(!(machine_stat & NOPOWER))
+			do_sparks(5, TRUE, src)
+			electrocute_mob(user, get_area(src), src, 1, TRUE) //zorp
+			close()
+		to_chat(user, span_notice("You start to cut [src] apart"))
+		if (W.use_tool(src, user, 15 SECONDS, volume = 75))
+			deconstruct(TRUE)
```

Comment:
should use /deconstruct_act here but also surplus white space

Reply:
not defined for this object